google/cloud/sql/connector/refresh_utils.py

```diff
@@ -181,7 +182,8 @@ async def _get_ephemeral(
     data = {"public_key": pub_key}
 
     if enable_iam_auth:
-        data["access_token"] = credentials.token
+        login_creds = _downscope_credentials(credentials)
```

Comment:
Can we add a comment here that says "downscope credentials refreshes them, so we don't need to do so here". I want to make sure it's not lost in terms of importance to create_ephemeral

Reply:
done